### DISCUSSION TITLE
Export for project overview

### DISCUSSION THREAD

**Discussion by flexminister** [kimai/kimai#2907] [Questions & Answers] [answered]
Is it generally possible to generate and export an evaluation for all projects with total and remaining hours as csv?

what i need is: "Reports > Project overview" + additional project information like comments, as csv

**Comment by kevinpapst** [comment, marked as answer]
Nope, currently not possible. 😞

**Reply by kevinpapst** [reply]
If you create a feature request in the Issues section, it might be added though.
